pom.xml

```diff
@@ -46,6 +46,12 @@
             <artifactId>bcpkix-jdk15on</artifactId>
             <version>1.64</version>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>5.10.0</version>
```

Comment:
Is there a reason why the version is 5.10? The newest version on maven central is 5.12 https://mvnrepository.com/artifact/org.mockito/mockito-core/5.12.0

Reply:
I looked up the version that is used in axm. I will update it to the latest. Thanks!